.env

```diff
@@ -1 +1,2 @@
 SASS_PATH="node_modules"
+REACT_APP_GITHUB_PERSONAL_ACCESS_TOKEN=d499f2f6cb18a22b8a67e44698521647f6f90171
```

Comment:
It is best practice not to commit an access token to GitHub. This should have been put in a env.local file.